DirectProgramming/Fortran/DenseLinearAlgebra/vectorize-vecmatmult/README.md

```diff
@@ -0,0 +1,224 @@
+# `Vectorize VecMatMult`
+
+In this sample, you will use the auto-vectorizer to improve the performance 
+of the sample application. You will compare the performance of the 
+serial version and the version that was compiled with the auto-vectorizer. 
+  
+| Optimized for                     | Description
+|:---                               |:---
+| OS                                | macOS* with Xcode* installed 
+| Hardware							   | Intel-based Mac*
+| Software                          | Intel&reg; oneAPI Intel Fortran Compiler (beta)
+| What you will learn               | Vectorization using Intel Fortran compiler
+| Time to complete                  | 15 minutes
+
+
+## Purpose
+The Intel® Compiler has an auto-vectorizer that detects operations in the application 
+that can be done in parallel and converts sequential operations 
+to parallel operations by using the 
+Single Instruction Multiple Data (SIMD) instruction set.
+
+For the Intel® compiler, vectorization is the unrolling of a loop combined with the generation of packed SIMD instructions. Because the packed instructions operate on more than one data element at a time, the loop can execute more efficiently. It is sometimes referred to as auto-vectorization to emphasize that the compiler automatically identifies and optimizes suitable loops on its own.
+
+Intel® Advisor can assist with vectorization and show optimization report messages with your source code. See [Intel Advisor][1] for details.
+[1]: https://software.intel.com/en-us/intel-advisor-xe "Intel Avisor"
```

Comment:
Change the link
https://software.intel.com/en-us/intel-advisor-xe
to
https://software.intel.com/content/www/us/en/develop/tools/advisor.html

Reply:
done.